docs/supported-components.human

```diff
@@ -0,0 +1,241 @@
+============================================================
+node version 10
+============================================================
+@hapi/hapi
+  17.9.0-18.4.1
+@hapi/vision
+  5.5.0-6.1.0
+amqplib
```

Comment:
[currently  documented](https://documentation.solarwinds.com/en/success_center/appoptics/content/kb/apm_tracing/nodejs/support-matrix.htm)
0.2.0-0.3.2, 0.4.1-0.4.2, 0.5.1-0.5.3, 0.5.5
This run results:
0.2.1-0.3.2, 0.4.1-0.5.3, 0.5.5-0.8.0